cmd/context/create.go

```diff
@@ -381,14 +381,22 @@ func (c ContextCommand) getUserContext(ctx context.Context, ns string) (*types.U
 			}
 		}
 
+		if utils.LoadBoolean(model.OktetoUseStaticKubetokenEnvVar) {
+			oktetoLog.Warning("Using static Kubernetes token due to env var: '%s'. This feature will be removed in the future. We recommend using a dynamic kubernetes token, to know more check out our documentation: https://www.okteto.com/docs", model.OktetoUseStaticKubetokenEnvVar)
```

Comment:
Could you post an image of how this message looks like in Warning and Information? I think it should be a warning because it's changing the behavior based on a env var but since we wanted to change it on context initialization, let's show both options and confirm with @rlamana

Reply:
@jLopezbarb 
<img width="1840" alt="Screenshot 2023-08-16 at 11 01 48" src="https://github.com/okteto/okteto/assets/2318450/293a39ac-abae-4c01-b521-3b387d6f2400">